embedding-store-filter-parsers/langchain4j-embedding-store-filter-parser-sql/src/main/java/dev/langchain4j/store/embedding/filter/builder/sql/LanguageModelSqlFilterBuilder.java

```diff
@@ -136,13 +138,13 @@ public Filter build(Query query) {
 
         String generatedSql = response.content().text();
 
-        String cleanedSql = clean(generatedSql);
+        String cleanedSql = generatedSql.trim();
```

Comment:
This is potentially breaking change: `clean(String)` is there to let users to extend this class and override the cleaning behavior (if required)

Reply:
Let's drop this and take #2294 then